src/pass/lower_intrin.cc

```diff
@@ -50,7 +50,23 @@ class IntrinInjecter : public IRMutator {
     // on ARM.
     if (const Broadcast* bcast = e.as<Broadcast>()) {
       if (const Cast* cast = bcast->value.as<Cast>()) {
-        if (cast->type.bits() == cast->value.type().bits() * 2) {
+        auto shouldSwap = [&]() {
```

Comment:
nit: code style, local variable need to be ```should_swap```?

Reply:
Done, my bad.